src/linear_algebra/general_linear_group.lean

```diff
@@ -186,11 +186,28 @@ lemma to_GL_pos_injective :
 (show function.injective ((coe : GL_pos n R → matrix n n R) ∘ to_GL_pos),
  from subtype.coe_injective).of_comp
 
+lemma coe_to_GL_pos_ext (g : special_linear_group n R) (i j : n) :
+  g.1 i j = (g : (GL_pos n R)).1.1 i j := rfl
```

Comment:
This looks like it's just a backwards version of `coe_GL_pos_coe_GL_coe_matrix` below, so I'd be inclined to remove it.

Reply:
Yes the .1's was just because after I merged with #12397 things had broken.